client/me/concierge/cancel/index.js

```diff
@@ -59,34 +60,47 @@ class ConciergeCancel extends Component {
 				);
 
 			default:
+				const disabledCancelling =
+					includes(
+						[ CONCIERGE_STATUS_CANCELLED, CONCIERGE_STATUS_CANCELLING ],
+						signupForm.status
+					) || ! appointmentDetails;
+
+				const disabledRescheduling =
+					signupForm.status === CONCIERGE_STATUS_CANCELLING || ! appointmentDetails;
+
 				return (
-					<Confirmation
-						description={ translate(
-							'You can also reschedule your session. What would you like to do?'
-						) }
-						title={ translate( 'Cancel your Concierge session' ) }
-					>
-						<Button
-							className="cancel__reschedule-button"
-							disabled={ signupForm.status === CONCIERGE_STATUS_CANCELLING }
-							href={ `/me/concierge/${ siteSlug }/${ appointmentId }/reschedule` }
-						>
-							{ translate( 'Reschedule session' ) }
-						</Button>
+					<div>
+						<QueryConciergeAppointmentDetails
+							appointmentId={ appointmentId }
+							scheduleId={ WPCOM_CONCIERGE_SCHEDULE_ID }
+						/>
 
-						<Button
-							className="cancel__confirmation-button"
-							disabled={ includes(
-								[ CONCIERGE_STATUS_CANCELLED, CONCIERGE_STATUS_CANCELLING ],
-								signupForm.status
+						<Confirmation
+							description={ translate(
+								'You can also reschedule your session. What would you like to?'
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'You can also reschedule your session. What would you like to do?' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).